source/openapi3/split.yaml

```diff
@@ -1847,12 +1759,12 @@ tags:
             "parent_ref": null,
             "type": "debit",
             "category": "payout_refund",
-            "created_at": "2016-12-07T23:15:00Z",
-            "matures_at": "2016-12-10T23:15:00Z",
+            "created_at": "2021-04-07T23:15:00Z",
+            "matures_at": "2021-04-10T23:15:00Z",
```

Comment:
Modernised! 😁

Reply:
Haha totally, there are so many of these so ill update the dates and schemas in another PR :)